lib/adapters/MochaAdapter.js

```diff
@@ -53,60 +52,69 @@ export default class MochaAdapter extends EventEmitter {
       parent = parent.parent
     }
 
-    var test = new Test(
-      testName,
-      suiteName
-    )
-    this.idDataMap[mochaTest.title] = test
+    var test
```

Comment:
I'd remove this and the `return test` later, and do 2 returns from the `if/else`.